README.md

```diff
@@ -3,11 +3,29 @@
 This repo reproduces the issue of inserting records into a MySQL/MariaDB using the [EntityFrameworkCore MySQL connector](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql);
 
 ### Requirements
-* dotnet core 2.1 SDK
+* [dotnet core 2.2 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.2)
```

Comment:
Actually the problem happens when it's run in a Lambda, there's nothing particularly special in that runtime, except for the fact that they natively support only .NET Core 2.1. I haven't tested yet using .NET Core 2. I can run a test with a custom runtime using .NET Core 2.2 which will support a higher version of the MySQL connector. But I am going to need some more time with that.
```suggestion
* [dotnet core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)
```

Reply:
Hi @AngelVenchev,

It sounds like you may be misunderstanding the problem that this `README` change is trying to solve. Your current code requires dotnet core 2.2 simply because the `TargetFramework` property is defined with the value `netcoreapp2.2` here:

https://github.com/AngelVenchev/max-scale-proxy-mysql-connector-problem-repro/blob/7e955bb5466b7128ec4840b2e3a8984192695268/MaxScaleProxyProblem/MaxScaleProxyProblem.csproj#L5

Currently, if you try to run the project without the dotnet core 2.2 SDK installed, then you get the following error:

```
> dotnet run
C:\Program Files\dotnet\sdk\2.1.700\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 2.2.  Either target .NET Core 2.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.2.
```

If you want to use dotnet core 2.1 instead, then it seems that you would have to change the `TargetFramework` property to the value `netcoreapp2.1`. See the following documentation page for the available values for the `TargetFramework` property:

https://docs.microsoft.com/en-us/dotnet/standard/frameworks

I don't really have a personal preference for using dotnet core 2.1 or 2.2. But if you want my colleagues and me to use this repository to try to reproduce your problem, then please make sure that the steps shown in the `README` actually work. My suggested change is simply trying to get the steps shown in the `README` to conform with your project's build configuration. If you don't want to change the dotnet core version in the `README`, then please change the dotnet core version in the build configuration.

Reply:
Point taken. Basically, the situation is that we're running .NET Core 2.1 on a Lambda since this is the maximum version allowed. I will change the target framework to be .NET Core 2.1 in this repo's code.
However, if the code works properly with .NET Core 2.2 I'd be happy to work and update the version we're currently targeting and deploying a custom runtime lambda that uses .NET Core 2.2 in order to fix the issue.

Reply:
I downgraded the version to version 2.1